Configuration/StandardSequences/python/RawToDigi_Data_cff.py

```diff
@@ -6,3 +6,10 @@
 ecalDigis.DoRegional = False
 #False by default ecalDigis.DoRegional = False
 
+# RPC Merged Digis 
+from Configuration.Eras.Modifier_run3_RPC_cff import run3_RPC
```

Comment:
@dilsonjd is this update only for data and not MC? Otherwise why in this configuration and not in RawToDigi_cff?

Reply:
If we run the data, this modification is necessary. Then, we understand that here is the correct place to implement it.

Reply:
@dilsonjd but is this supposed to be activated *ONLY* on data? Do we have a test were this code is practically exercised? Your tests in EventFilter do not seem dependent on this run3 Era.

Reply:
@fabiocos Yes, it is to be actived ONLY on data. 
The test was done running over the Run2018A dataset using the eras.Run2_2018 and adding eras.run3_RPC into the conditions of the process. You can find the configs here  /afs/cern.ch/work/d/dilson/public/toTestRun3RPCData/

Maybe, the modifier_run3_RPC must be add to Era_Run3_cff.py. What do you think?

Reply:
this kind of modifications should be done in the RawToDigi_cff , common to data and simulation.

Reply:
The configuration in the Raw2Digi_Data is due to the two different strategies needed for the MC and Data approach.